coresdk/src/coresdk/types.h

```diff
@@ -43,6 +43,14 @@ enum font_style
     UNDERLINE_FONT   = 4
 };
 
+/// Provides an overload that allows combining font styles with the bit or 
+/// operator.
+/// Source: http://stackoverflow.com/a/1448478
+inline font_style operator|(font_style a, font_style b)
```

Comment:
@macite Yeah not sure about this one. Possible but may need to have a look at how Headerdoc parses it. But ideologically I'm not sure it's a good idea only because students won't use it or they won't understand it at all...